src/banman.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2017 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#ifndef BITCOIN_BANMAN_H
+#define BITCOIN_BANMAN_H
+
+#include "addrdb.h"
+#include "fs.h"
+#include "sync.h"
+
+// NOTE: When adjusting this, update rpcnet:setban's help ("24h")
+static const unsigned int DEFAULT_MISBEHAVING_BANTIME = 60 * 60 * 24;  // Default 24-hour ban
+
+class CClientUIInterface;
+class CNetAddr;
+class CSubNet;
+
+class BanMan
+{
+public:
+    // Denial-of-service detection/prevention
```

Comment:
Should this be a class-level comment? (Like placed right above `class BanMan`)

Reply:
Sure

Reply:
BanMan still needs class-level comment.